extensions/amp-video/1.0/base-element.js

```diff
@@ -95,6 +95,10 @@ VideoBaseElement['props'] = {
   'loop': {attr: 'loop', type: 'boolean'},
   'noaudio': {attr: 'noaudio', type: 'boolean'},
   'poster': {attr: 'poster'},
+};
+
+/** @override */
+VideoBaseElement['children'] = {
```

Comment:
`['children']` def has been combined into `['props']` def, so these lines should be excluded.